cmd/ci-operator/main.go

```diff
@@ -665,9 +665,6 @@ func (o *options) initializeNamespace() error {
 		})
 	}
 
-	if len(o.secrets) > 0 {
-		log.Printf("Populating secrets for test")
```

Comment:
Why?

Reply:
Not actually useful, was just noise.  We already print when we populate a secret.